src/modules/navigator/geofence_params.c

```diff
@@ -66,13 +66,13 @@ PARAM_DEFINE_INT32(GF_ACTION, 1);
 /**
  * Geofence altitude mode
  *
- * Select which altitude reference should be used
- * 0 = WGS84, 1 = AMSL
+ * Select which altitude source should be used
+ * 0 = Estimated altitude (GPS/IMU etc.), 1 = Barometer altitude.
  *
  * @min 0
  * @max 1
- * @value 0 WGS84
- * @value 1 AMSL
+ * @value 0 Estimated altitude
```

Comment:
The change makes it not better. "Estimated" doesn't mean anything. PX4 only knows about AMSL and barometric altitude. This needs proper inspection.

Reply:
The setting 0 is AMSL provided by GPS. The setting 1 is barometric AMSL, but with the standard atmosphere. So it might be above or below sea level.

Reply:
@LorenzMeier "Estimated" is the information provided by Daniel in linked discussion. I assumed it was the EKF altitude (from GPS and IMU). I've updated in line with what I understand you want. If this is not correct, please edit directly.